### DISCUSSION TITLE
TOS(TERMS OF SERVICE)  in GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#116927] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by kairusamaa** [comment, marked as answer]
Linking to platforms like Ko-fi and Patreon for financial support seems acceptable based on GitHub's TOS
Additionally, regardless of whether an Action is using self-hosted runners, Actions should not be used for:
cryptomining;
disrupting, gaining, or attempting to gain unauthorized access to, any service, device, data, account, or network (other than those authorized by the [GitHub Bug Bounty program](https://bounty.github.com/));
the provision of a stand-alone or integrated application or service offering the Actions product or service, or any elements of the Actions product or service, for commercial purposes;
any activity that places a burden on our servers, where that burden is disproportionate to the benefits provided to users (for example, don't use Actions as a content delivery network or as part of a serverless application, but a low benefit Action could be ok if it’s also low burden); or
if using GitHub-hosted runners, any other activity unrelated to the production, testing, deployment, or publication of the software project associated with the repository where GitHub Actions are used.
